RHEL/6/input/profiles/ftp-server.xml

```diff
@@ -1,6 +1,6 @@
 <Profile id="ftp-server" extends="server">
-<title override="true">ftp</title>
-<description override="true">This profile is for FTP servers.</description>
+<title override="true">FTP Server Profile (vsftpd)</title>
+<description override="true">This is a profile for the vsftpd FTP server</description>
```

Comment:
Well crap, you're right. Will merge this and I'll open a separate PR for complete sentences in our profile descriptions.